adding_species_manuscript.tex

```diff
@@ -471,26 +471,32 @@ \subsection*{Basic setup for chromosome-level simulations}
   \textbf{A chromosome-level genome assembly}, which consists of a list of chromosomes or scaffolds and their lengths. 
   Having a good quality assembly with complete chromosomes, or at least very long scaffolds, 
   is necessary if chromosome-level population genomic simulations are to reflect the genomic architecture of the species.
-  Currently, the number of species with complete chromosome-level assemblies is small,
-  but we expect this number to dramatically increase in the near future due to genome initiatives 
-  such as the Earth Biogenome \citep{Lewin2022} and its affiliated project networks (e.g.,
-  Vertebrate Genomes \citep{Rhie2021}, 10,000 Plants \citep{Cheng2018}).
-  Furthermore, the development of new long-read sequencing technologies
-  \citep{Amarasinghe2020,Amarasinghe2021} and concomitant advances in assembly pipelines
-  \citep{Chakraborty2016} are likely to boost these initiatives. 
-  When expanding the \stdpopsim catalog, we decided to focus on species with near-complete 
-  chromosome-level genome assemblies (i.e., close to one contig per chromosome).
-  This restriction was set mainly because species with less complete genome builds 
-  typically do not have genetic maps or good estimates of recombination rates, 
-  making chromosome-level simulation much less useful. 
-  Therefore, the utility of adding such species to the catalog does not justify the 
-  maintenance and storage burden incurred by the large number of contigs in these partial assemblies (see also discussion below).
+  When expanding the \stdpopsim catalog during the ``Growing the Zoo" hackathon,
```

Comment:
Several people noted the redundancy between the discussion here and the one we have in the 'missing pieces' section,. So I decided to move all the discussion here, and make only a brief mention in the later section.